calibrators/marker_radar_lidar_calibrator/README.md

```diff
@@ -0,0 +1,169 @@
+# marker_radar_lidar_calibrator
+
+A tutorial for this calibrator can be found [here](../docs/tutorials/marker_radar_lidar_calibrator.md)
+
+## Purpose
+
+The package `marker_radar_lidar_calibrator` allows extrinsic calibration between radar and 3d lidar sensors used in autonomous driving and robotics.
+
+Currently, the calibrator only supports radars whose detection interface includes distance and azimuth angle, but do not offer elevation angle. For example, ARS 408 radar can be calibrated with this tool. Also, note that the 3d lidar should have a resolution that is high enough to scan several points on the [radar reflector](#radar-reflector) (calibration target).
+
+## Inner-workings / Algorithms
+
+The calibrator is designed to estimate the transformation between radar and lidar sensors. It starts by estimating the center points of reflectors within lidar pointclouds and radar objects and then aligns these points for precise matching. An SVD-based and a yaw-only rotation estimation algorithm are applied to these matched points to estimate the rigid transformation between sensors.
+
+Specifically, the calibration process consists of four primary steps: constructing a background model, extracting the foreground to detect reflectors, matching and filtering lidar and radar detections, and finally estimating rigid transformation between radar and lidar sensors.
```

Comment:
missing  `the`

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/d3963ac2951442d27d3e9a95529579239fa403db